cpp/ps0003/ps0003_sol.cpp

```diff
@@ -4,14 +4,17 @@ using namespace std;
 
 int main()
 {
-    short s ;
-    cin >> s ;
-     if ( s % 2 == 0 ) {
+    // prompting user for number 2/2
+    short s;
+    cin >> s;
+
+    // checking if the number is odd or even 3/3
+    if (s % 2 == 0) {
         cout << "Number is even \n" ;
-     } else {
+    } else {
         cout<< "Number is odd \n" ;
-     }
+    }
 
     return 0;
 }
-
+// great work 😎 9/10 ⭐

```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code